src/main/java/model/lotto/Lottos.java

```diff
@@ -0,0 +1,46 @@
+package model.lotto;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import model.lottonumber.LottoNumbers;
+import model.lottonumber.RandomLottoNumbersGenerationStrategy;
+import model.result.RateOfReturn;
+import model.winningnumber.WinningLottoNumberDTO;
+
+public class Lottos {
+	private List<Lotto> lottoStorage;
+
+	public Lottos(LottoCount lottoCount) {
+		this.lottoStorage = store(lottoCount);
+	}
+
+	private List<Lotto> store(LottoCount lottoCount) {
+		List<Lotto> lottos = new ArrayList<>();
+
+		while (lottoCount.haveRemainToMake()) {
+			lottoCount.countMadeLotto();
+			lottos.add(new Lotto(LottoNumbers.from(new RandomLottoNumbersGenerationStrategy())));
+		}
+
+		return lottos;
```

Comment:
스트림을 사용하신다면 아래와 같은 방식도 가능합니다.

```suggestion
        LottoNumbersGenerationStrategy strategy = new RandomLottoNumbersGenerationStrategy();
        
        return IntStream.range(0, lottoCount.getCount())
                .mapToObj((i) -> new Lotto(LottoNumbers.from(strategy)))
                .collect(Collectors.toList());
```